server/schedule/operator/create_operator.go

```diff
@@ -217,6 +217,13 @@ func CreateScatterRegionOperator(desc string, ci ClusterInformer, origin *core.R
 		Build(0)
 }
 
+const opDescLeaveJointState = "leave-joint-state"
+
+// NewLeaveJointStateOperator  creates an operator that let region leave joint state with default desc.
+func NewLeaveJointStateOperator(ci ClusterInformer, origin *core.RegionInfo) (*Operator, error) {
```

Comment:
What is the difference between `NewLeaveJointStateOperator` and `CreateLeaveJointStateOperator`?

Reply:
Similar to the `CreateXXXOperator`, the user can define the desc in `CreateLeaveJointStateOperator`, such as in the test. While `NewLeaveJointStateOperator ` use a constant as the desc so we can easily find the operator that was created by `leave joint state`